cumulusci/core/config/project_config.py

```diff
@@ -605,3 +622,7 @@ def open_cache(self, cache_name):
         with open_fs_resource(self.cache_dir / cache_name) as cache_dir:
             cache_dir.mkdir(exist_ok=True, parents=True)
             yield cache_dir
+
+
+class RemoteProjectConfig(BaseConfig):
+    project: dict
```

Comment:
Why do we need a separate RemoteProjectConfig?

Reply:
It seems to have a single property. What other type would be preferable?

Reply:
Okay I'll change it to return a BaseProjectConfig. Why is BaseProjectConfig called that when there don't seem to be classes which inherit from it? Are we using "Base" to mean something different than Base Class?

Reply:
Actually we need RemoteProjectConfig because BaseProjectConfig has initialization code that expects a local config.

But I can broaden RemoteProjectConfig's list of properties to match BaseProjectConfig: 9f8860355

Reply:
@prescod what are you hitting exactly? I think BaseProjectConfig avoids trying to read local config if a config dict was passed in when it's initialized.

Reply:
For whoever reads this in the future: the root of the confusion is that BaseProject takes config as its first init argument and BaseProjectConfig takes it as its second. So config was being defaulted to None and then {} and then attempted to be loaded.

Reply:
Fixed in 40254a0ad9d505b65131346980ceea4588b11067